webrtc/videoroom/lib/videoroom/ws.ex

```diff
@@ -0,0 +1,62 @@
+defmodule VideoRoom.WS do
+  @behaviour :cowboy_websocket
+
+  # Server API
+  @impl true
+  def init(req, state) do
+    {:cowboy_websocket, req, state}
+  end
+
+  @impl true
+  def websocket_init(_state) do
+    {:ok, %{}}
+  end
+
+  @impl true
+  def websocket_handle({:text, "keep_alive"}, state) do
+    {:ok, state}
+  end
+
+  @impl true
+  def websocket_handle({:text, msg}, state) do
+    msg = Poison.decode!(msg)
+
+    msg =
+      case msg["event"] do
+        "start" -> {:new_peer, self()}
```

Comment:
Is it a good idea to identify peers by WS pid? On browser side we are sending `keep_alives` to keep our WS alive. Maybe it would be better to generate some id and use it instead of WS pid. In such scenario we don't have to send keep alives from browser. If WS terminates browser can create a new one and send its request using id we previously assigned to it. If this sounds better we can create issue for this.

Reply:
Keeping websocket open is useful, for example to determine that somebody has left, and generally makes interaction with the frontend much easier. Also, that's what everyone does.